### ISSUE TITLE
Node 14 not supported

### ISSUE THREAD

**Issue by Stegosource** [architect/architect#780]
**Describe the issue**
@architect/sandbox seems to hit an issue when running Node 14. I dont think this is a major issue as AWS does not yet support 14, but I thought it might be good to report here so it's known.

**Steps to reproduce**
1. Install and use Node 14
2. Run project with `npm start`

**Expected behavior**
Project should run

**Desktop**
Please complete the following information (if appropriate):
 - Ubuntu 20
- Node 14
- sandbox 1.9.3

**Additional context**
To be specific, the package that fails is node-watch, so not something specific to Architect.

**Comment by ryanblock** [comment]
Austin, thanks! That's great to know. A couple things:

- Sandbox's job is to emulate AWS production environments; as you mentioned, Node.js is not supported in Lambda yet – when it is, we will ensure first-class support
- Working locally, Node.js 14 is "current" but not LTS – meaning it is not recommended for production workloads. Per the Node.js TSC:
```
As always, corporate users should wait to upgrade their production deployments until October when Node.js is promoted to LTS. However, now is the best time to start testing applications with Node.js 14, and try out new features.
```
- Given the fact that Architect is designed to enable production workloads, it follows that we will not be fully supporting Node.js 14 until it reaches LTS

All that said, `node-watch` failing is excellent to note, and I'll start working to ensure they are prepared in time for 14 to hit LTS later this year.

**Comment by Stegosource** [comment]
Right. I figured this one would get closed right away. Hope that's ok that I opened it. If not, let me know a better way to report these sorts of things.

**Comment by ryanblock** [comment]
It's totally ok! This is on the radar now and other folks can find it – and like I mentioned, knowing `node-watch` is borken is valuable. While we're on that topic, do you know what was failing with `node-watch`?

**Comment by Stegosource** [comment]
```
fs.js:1441
    throw new ERR_FEATURE_UNAVAILABLE_ON_PLATFORM('watch recursively');
    ^

TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
```

**Comment by ryanblock** [comment]
Hmm, what else can you tell me about your platform? I'm unable to replicate that error:
```

**Comment by Stegosource** [comment]
What else do you need to know. I mentioned my OS is Ubuntu 20. Besides that, Im not sure what else could be related. NPM version is 6.14.4.

**Comment by ryanblock** [comment]
Woo! https://github.com/yuanchuan/node-watch/blob/master/Changelog.md#064

Will be fixed in Sandbox 1.10.0 + Arc 6.4.0 today!
